tests/components/modbus/test_light.py

```diff
@@ -338,6 +495,29 @@ async def test_service_light_update(hass: HomeAssistant, mock_modbus_ha) -> None
         blocking=True,
     )
     assert hass.states.get(ENTITY_ID).state == STATE_ON
+    mock_modbus_ha.read_holding_registers.return_value = ReadResult(
+        [brightness_input, color_temp_input]
+    )
+
+    await hass.services.async_call(
+        HOMEASSISTANT_DOMAIN,
+        SERVICE_UPDATE_ENTITY,
+        {
+            ATTR_ENTITY_ID: ENTITY_ID,
+        },
+        blocking=True,
+    )
+    if hass.states.get(ENTITY_ID).attributes.get(ATTR_BRIGHTNESS):
+        assert (
+            hass.states.get(ENTITY_ID).attributes.get(ATTR_BRIGHTNESS)
+            == expected_brightness
```

Comment:
```suggestion
    assert (expected_brightness is None or
        hass.states.get(ENTITY_ID).attributes.get(ATTR_BRIGHTNESS)
        == expected_brightness
```
that way we tie this to the test input and not to the state (which may be wrong)

Reply:
same for the color temp bellow